src/tokens/saga.ts

```diff
@@ -248,8 +248,10 @@ export function* watchAccountFundedOrLiquidated() {
     // we reset the usd value of all token balances to 0 if the exchange rate is
     // stale, so it is okay to use stale token prices to monitor the account
     // funded / liquidated status in this case
+    const supportedNetworkIds = getSupportedNetworkIdsForTokenBalances()
```

Comment:
is there a chance that this can be invoked before statsig is initialized? If so, one scenario I can think of is, let's say a user has only ETH assets. And if this is invoked before statsig is initialized, we'd end up checking only celo assets (the default value for the dynamic config) and potentially fire an account_liquidated event.

Reply:
@satish-ravi hmm yeah I'm not sure, do you know when statsig is initialized vs. when saga stuff starts tracking? I'm still learning about how all of the saga/event stuff works.

Reply:
I'd start at the rootSaga in src/redux/sagas.ts. It looks like all other sagas are invoked after appInit, which is what initializes statsig, but would be good to confirm by adding some logs on the network Ids used and making sure the scenario doesn't happen

Reply:
sounds good I can test out locally and make sure that both network IDs are always included (for my case where I am in the multichain beta)

Reply:
> is there a chance that this can be invoked before statsig is initialized? If so, one scenario I can think of is, let's say a user has only ETH assets. And if this is invoked before statsig is initialized, we'd end up checking only celo assets (the default value for the dynamic config) and potentially fire an account_liquidated event.

@satish-ravi I tested locally and it seems like statsig was always initialized before the funded/liquidated functions were invoked, so should be okay.